src/js/mods/modloader.js

```diff
@@ -187,16 +198,25 @@ export class ModLoader {
             this.modLoadQueue.push({
                 modClass,
                 meta,
+                path,
             });
         };
 
         mods.forEach(modCode => {
+            path = modCode.slice(0, modCode.indexOf("\n")).replace("// ", "");
+            if (path.endsWith(".disabled")) {
+                modCode =
+                    `
+                           let $old_shapez = structuredClone(shapez);
```

Comment:
this seems extremely unreliable (and expensive); `structuredClone` isn't magic https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Structured_clone_algorithm

and of course it doesn't handle alterations to the global namespace and APIs, even if mods usually shouldn't touch that